### DISCUSSION TITLE
What am I supposed to do with the provided LUT file archives?

### DISCUSSION THREAD

**Discussion by n0valis** [awawa-dev/HyperHDR#389] [General]
... was the question I asked myself after I downloaded LUT_MS2109_1_2.tar.xz from https://github.com/awawa-dev/HyperHDR/releases/tag/v18.0.0.0
And I took me while to find it out by reading posts from other users.
Maybe it's just me but I couldn't find an answer to that, neither here on github, nor on the blog page.
Otherwise please post the link here please.
So I just wanted to provide the solution for someone who might find that helpful:

Extract the archive (tar -xf LUT_MS2109_1_2.tar.xz) and place the lut_lin_tables.3d file in your ~/.hyperhdr directory
If you want to know wheter it works you can check the logs (http://your_ip:8090/#logs) and you should find a line like this:
(Grabber.cpp:445) LUT file found: /home/bob/.hyperhdr/lut_lin_tables.3d

**Comment by chesterflaps** [comment]
Is it possible to do this via SSH or do you have to do this with the sd card plugged into your computer to put it in the folder?
